rawpointer_test.go

```diff
@@ -232,3 +232,39 @@ func TestRawPointerTotalListSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLandingPadNearPointer(t *testing.T) {
```

Comment:
This test doesn't really help me understand what failed with the previous implementation.  Instead, can you write this as a table test of:

```go
tests := []struct{
  far     rawPointer
  tag     rawPointer
  landing rawPointer
}{
  // ...
}
```

The test as written is halfway between a unit test and an integration test (usually a bad thing): it checks results by using other functions instead of testing just this function.

It would be nice to have a small integration test as well.  I know far pointers don't have as great test coverage, so it would help to have more.